components/MediaPlayer.vue

```diff
@@ -113,62 +261,92 @@ const {
-        @click.stop="queue.shuffle()"
-      >
-        <NuxtIcon name="icon.shuffle" class="text-2xl" />
-      </button>
-    </div>
+      <hr
+        class="absolute left-0 right-0 top-[450px] border-label-separator pb-4"
+      />
+      <!-- add forward function -->
+      <div class="flex justify-between pb-3">
+        <div class="text-label-3">queue</div>
+        <div class="flex gap-2">
+          <NuxtIcon name="icon.shuffle" filled class="text-2xl" />
+          <NuxtIcon name="icon.repeat" filled class="text-2xl" />
+        </div>
+      </div>
+      <ul class="scrollbar-hide max-h-20 overflow-y-scroll">
+        <li
+          v-for="(item, i) in queue"
+          :key="i"
+          :class="queue.index === i ? 'bg-tint' : ''"
+          @click="queue.index = i"
+          class="rounded-xl"
+        >
+          <div class="flex justify-between p-1">
+            <div class="titles">
+              <div>{{ item.meta?.title || item.title }}</div>
+              <div class="text-sm">
+                <span
+                  v-if="item?.meta?.artist"
+                  class="truncate leading-snug text-label-2 dark:text-label-dark-2"
+                  :title="item?.meta?.artist"
+                >
+                  {{ item.meta?.artist }}
+                </span>
+                <span v-if="item?.meta?.artist && item?.meta?.album"> - </span>
+                <span
+                  v-if="item?.meta?.album"
+                  class="truncate leading-snug text-label-2 dark:text-label-dark-2"
+                  :title="item?.meta?.album"
+                >
+                  {{ item.meta?.album }}
+                </span>
+              </div>
+            </div>
+            <div class="icons flex gap-2">
+              <NuxtIcon name="options" filled class="text-2xl" />
+              <NuxtIcon
+                v-if="queue.index === i"
+                name="icon.playing (animation)"
+                filled
+                class="text-2xl"
+              />
+            </div>
+          </div>
 
-    <ul class="max-h-20 overflow-y-scroll">
-      <li
-        v-for="(item, i) in queue"
-        :key="i"
-        :class="queue.index === i ? 'bg-tint' : ''"
-        @click="queue.index = i"
-      >
-        {{ item.meta?.title || item.title }}
-      </li>
-    </ul>
-  </div>
+          <hr
+            v-if="!(queue.index - i === 1 || queue.index - i === 0)"
+            class="border-label-separator"
+          />
+        </li>
+      </ul>
+    </div>
+  </transition>
 </template>
+<style>
+.shadow-player {
+  box-shadow: 0px 4px 12px 0px rgba(0, 0, 0, 0.05),
+    0px 1px 4px 0px rgba(0, 0, 0, 0.05), 0px 0px 0px 1px rgba(0, 0, 0, 0.05);
+}
+
+/* For Webkit-based browsers (Chrome, Safari and Opera) */
+.scrollbar-hide::-webkit-scrollbar {
```

Comment:
@sclausendk I'm quite against just hiding the scroll bar without giving the user anything else to go by. In apps this might be a common way to go, but in the web, I find it quite disturbing on a list. What's your opinion on this?